src/geophires_x/CylindricalReservoir.py

```diff
@@ -68,6 +70,7 @@ def __init__(self, model: Model):
         )
         self.Length = self.ParameterDict[self.Length.Name] = floatParameter(
             "Cylindrical Reservoir Length",
+            value=4.0,
```

Comment:
Remove - only `DefaultValue` should be specified, not `value` (which is redundant with `DefaultValue` in this case anyway)